src/features/pastelDB/pastelDBLib.ts

```diff
@@ -113,13 +129,22 @@ export const createDatabase = async (): Promise<Database> => {
   const SQL = await initSqlJS()
 
   try {
-    const filebuffer: Buffer = await readSqliteDBFile()
-    return new SQL.Database(filebuffer)
+    const filebuffer: Buffer | null = await readSqliteDBFile()
+    if (filebuffer && filebuffer.length) {
+      const db = new SQL.Database(filebuffer)
+
+      // check database valid.
+      getLastIdFromDB(db, 'walletinfo')
+
+      return db
+    }
   } catch (error) {
-    const newdb: Database = new SQL.Database()
-    await createTables(newdb)
-    return newdb
+    console.log(`pastelDB createDatabase error: ${error}`)
```

Comment:
We already have installed the `electron-log` package. Can we use it to log? 

https://www.npmjs.com/package/electron-log